generator/src/components/GasSpeciesDetail.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { connect } from "react-redux";
+import { getGasSpecies } from "../redux/selectors";
+import PropertyList from "./PropertyList";
+
+const GasSpeciesDetail = ({ species, detailSpecies, setDetailSpecies }) => {
+  const handleClick = (e) => {
+    e.preventDefault();
+    var newDetailSpecies = { ...detailSpecies };
```

Comment:
While it doesn't really matter, we should use `let` here instead. `var` can have issues with something called hoisting and can outlive it's local scope while `let` (or `const`) can't. So better practice is to use `let` and `const` where possible and `var` only if we want a global variable.

Reply:
Good to know! Changed.